### PR TITLE
FE-637

### PR DESCRIPTION
**Pull request by ipanagiv**

### CODE REVIEW

File: docs/rewards/rewards-troubleshooting.mdx [hunk 1]
```diff
@@ -22,8 +22,8 @@ Once you've completed the above steps, you'll be all set to claim your rewards.
 
 ## No Station Data
 
-It seems that your station sent no data at all. 
-That either means that your station was offline or not connected to the network. 
+It seems that your station did not transmit any data, which may indicate that it was either offline or not connected to the network.
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:25]: any data --> any data at all

File: docs/rewards/rewards-troubleshooting.mdx [hunk 2]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
 
 ## Sensor Problems
 
-Weather data packets failed to pass our quality checks. We evaluate all data packets sent by our weather stations using our QoD (Quality of Data) algorithm. You can read more about this algorithm [here](https://docs.weatherxm.com/project/quality-of-data).
-There are several reasons that you may experience this error, which includes:
+We have discovered that the weather data packets received, did not meet our quality standards. 
+To ensure the accuracy of the information sent by our weather stations, we evaluate all data packets using our Quality of Data (QoD) algorithm. For more information about this algorithm, please refer to this [QoD](/project/quality-of-data).
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:67]: 1. Let's make `Quality of Data (QoD) algorithm` hyperlinked pointing to the docs algorithm.
2. to this [QoD] ---> [here]

File: docs/rewards/rewards-troubleshooting.mdx [hunk 3]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
 
 ## Sensor Problems
 
-Weather data packets failed to pass our quality checks. We evaluate all data packets sent by our weather stations using our QoD (Quality of Data) algorithm. You can read more about this algorithm [here](https://docs.weatherxm.com/project/quality-of-data).
-There are several reasons that you may experience this error, which includes:
+We have discovered that the weather data packets received, did not meet our quality standards. 
+To ensure the accuracy of the information sent by our weather stations, we evaluate all data packets using our Quality of Data (QoD) algorithm. For more information about this algorithm, please refer to this [QoD](/project/quality-of-data). 
+
+Following are the errors and the appropriate actions you can take to resolve the issues:
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:69]: Let's rephrase it to something like:

> Below are some of the possible errors and the appropriate actions you can take to resolve the issues:

File: docs/rewards/rewards-troubleshooting.mdx [hunk 4]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
+
+#### Out of bounds data
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:71]: Out of bounds data  ---> Out of Bounds Data

File: docs/rewards/rewards-troubleshooting.mdx [hunk 5]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
 
-Weather data packets failed to pass our quality checks. We evaluate all data packets sent by our weather stations using our QoD (Quality of Data) algorithm. You can read more about this algorithm [here](https://docs.weatherxm.com/project/quality-of-data).
-There are several reasons that you may experience this error, which includes:
+We have discovered that the weather data packets received, did not meet our quality standards. 
+To ensure the accuracy of the information sent by our weather stations, we evaluate all data packets using our Quality of Data (QoD) algorithm. For more information about this algorithm, please refer to this [QoD](/project/quality-of-data). 
+
+Following are the errors and the appropriate actions you can take to resolve the issues:
+
+#### Out of bounds data 
+OBC, means that a sensor sends a completely irrational value, which implies to a faulty sensor. It is recommended that you observe it for a few days before reaching out to our [support team](https://support.weatherxm.com)  for sensor replacement.
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:72]: The user doesn't know what "OBC" is, nor he should be aware of that. Let's remove it start the sentence accordingly.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 6]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
+
+#### Spikes on data
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:74]: Spikes on data ----> Spikes on Data

File: docs/rewards/rewards-troubleshooting.mdx [hunk 7]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
+
+Following are the errors and the appropriate actions you can take to resolve the issues:
+
+#### Out of bounds data 
+OBC, means that a sensor sends a completely irrational value, which implies to a faulty sensor. It is recommended that you observe it for a few days before reaching out to our [support team](https://support.weatherxm.com)  for sensor replacement.
+
+#### Spikes on data 
+You should first check your station if it is optimally deployed and then you should contact [support ](https://support.weatherxm.com).
+
+#### Constant values
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:77]: Constant values  ----> Constant Values

File: docs/rewards/rewards-troubleshooting.mdx [hunk 8]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
+You should first check your station if it is optimally deployed and then you should contact [support ](https://support.weatherxm.com).
+
+#### Constant values 
+You should first check your station if it is optimally deployed and then you should contact [support ](https://support.weatherxm.com).
 
-- Out of bounds data
-- Spikes on data 
-- Constant values
-- Frozen sensor
+#### Frozen sensor
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:80]: Frozen sensor ---> Frozen Sensor

File: docs/rewards/rewards-troubleshooting.mdx [hunk 9]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
+We have discovered that the weather data packets received, did not meet our quality standards. 
+To ensure the accuracy of the information sent by our weather stations, we evaluate all data packets using our Quality of Data (QoD) algorithm. For more information about this algorithm, please refer to this [QoD](/project/quality-of-data). 
+
+Following are the errors and the appropriate actions you can take to resolve the issues:
+
+#### Out of bounds data 
+OBC, means that a sensor sends a completely irrational value, which implies to a faulty sensor. It is recommended that you observe it for a few days before reaching out to our [support team](https://support.weatherxm.com)  for sensor replacement.
+
+#### Spikes on data 
+You should first check your station if it is optimally deployed and then you should contact [support ](https://support.weatherxm.com).
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:75]: Let's add this at the end:

> Take a look at our out our  [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 10]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
+
+#### Constant values 
+You should first check your station if it is optimally deployed and then you should contact [support ](https://support.weatherxm.com).
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:78]: Let's add this at the end:

> Take a look at our out our  [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 11]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
 
-- Out of bounds data
-- Spikes on data 
-- Constant values
-- Frozen sensor
+#### Frozen sensor
+You should remove excess snow or ice from the weather station, and make sure wind cups and wind vanes rotate freely.
+
+
+Take a look at our out our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly.
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:84]: Not needed here, let's remove it from here as we need to add it respectively above based on my comments.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 12]
```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
+
+:::tip
+It is recommended that you observe it for a few days before reaching out to our [support team](https://support.weatherxm.com) if you encounter any errors.
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:87]: "if you encounter any errors" let's move this at the start of the sentence.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 13]
```diff
@@ -117,19 +133,16 @@ If the battery is low, the connection between your station and the M5 gateway wi
 
 ## Bad Station Deployment
 
-It seems like your station needs some improvement in terms of how and where it is deployed.
-Check out our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) for some inspiration on how and where you should install your weather station.
+It appears that your weather station could benefit from some improvements in terms of its deployment. We recommend that you take a look at our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) for inspiration on how and where to install your station
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:136]: Add a dot "." at the end of the sentence.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 14]
```diff
@@ -161,8 +187,8 @@ During the beta phase and while the project is still on a test-net, the Proof of
 
 ## User Relocation Penalty
 
-It seems like you used the relocation feature and you edited the location of your weather station through the app.
-Please wait 7 days and your weather station will start earning rewards again.
+Our [Proof of Location (PoL) mechanism](/project/proof-of-location) reported a relocation event. 
+It seems like you used the relocation feature and you edited the location of your weather station through the app. Please wait 7 days and your weather station will start earning rewards again.
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:191]: Let's add this at the end or in a new line:

> WeatherXM places great importance on station continuity, and discourage relocations therefore It is important for the Network that a station remains at the same location throughout its lifetime.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 15]
```diff
@@ -145,12 +158,25 @@ Please make sure to check the troubleshooting guides depending on your weather s
 
 ## Location Not Verified
 
-It seems that your location is different from the one you set while claiming your station.
-Please verify your location through the mobile app (Station details -> Settings -> Edit Location)
+It appears that the location of your weather station is different from the one you had originally set while claiming it. 
+Please verify your location through the mobile app. (See the screenshots below for guidance)
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:162]: "(See the screenshots below for guidance)" is not needed IMO as the screenshots are right below the text so the user will see them either way.

**Review by PavlosTze — CHANGES_REQUESTED** [review]
Good job @ipanagiv , I have some minor comments only.

**Review by PavlosTze — APPROVED** [review]

File: docs/rewards/rewards-troubleshooting.mdx [hunk 16]
```diff
@@ -13,7 +13,7 @@ A guide to interpreting mobile app error messages, understanding their meanings,
 
 It seems that you haven't connected a wallet to your account yet. Please note that in order to receive rewards, you must connect a wallet address. Here's what you need to do:
 
-1. First, learn [how to add or edit a wallet address](/wallet/add-edit-wallet-address).
+1. Learn [how to add or edit a wallet address](/wallet/add-edit-wallet-address).
 2. Then, follow the instructions for [joining Arbitrum Sepolia and see your $WXM tokens](/wallet/how-to-join-arbitrum-and-see-your-tokens#add-arbitrum-sepolia-network-to-metamask).
 
 Once you've completed the above steps, you'll be all set to claim your rewards. Check back tomorrow and see "[How to Claim Rewards](/project/rewards-claiming)" for further guidance.
```
Comment (NikosTsiligaridis) [inline docs/rewards/rewards-troubleshooting.mdx:19]: "Check back tomorrow" could be something more descriptive that informs the users that once he configured his wallet he will see rewards in the next run which happens daily.

"Check back tomorrow" gives the impression that we're going to add something new to the docs tomorrow :)

File: docs/rewards/rewards-troubleshooting.mdx [hunk 17]
```diff
@@ -117,19 +135,16 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
 <hr/>
 
 ## No location Data
 
-Your station did not report any GPS location and was suspended from rewards because it didn't send any GPS data.
-
-Please make sure to check the troubleshooting guides depending on your weather station before contacting our [support team](https://support.weatherxm.com). 
+Your weather station was suspended from rewards because it did not report GPS data. Please check the troubleshooting guides for your station below, before contacting our support team.[support team](https://support.weatherxm.com). 
 
 ### For the WS-1000 (WiFi/M5 version)
```
Comment (NikosTsiligaridis) [inline docs/rewards/rewards-troubleshooting.mdx:149]: `For the M5 (WG1000 - WiFi gateway)`

M5 is the friendly name of this product, WG1000 is the SKU (WS1000 is the weather station).
We might as well start using the more up-to-date terminology which is also in line with the new products that are coming. Eg when the new gateway comes (next product - soon - you didn't hear it from me) there will no more be a single "wifi version", but there will be wifi gateways

File: docs/rewards/rewards-troubleshooting.mdx [hunk 18]
```diff
@@ -63,21 +63,39 @@ If the battery is low, the connection between your station and the M5 gateway wi
 
 ## Sensor Problems
 
-Weather data packets failed to pass our quality checks. We evaluate all data packets sent by our weather stations using our QoD (Quality of Data) algorithm. You can read more about this algorithm [here](https://docs.weatherxm.com/project/quality-of-data).
-There are several reasons that you may experience this error, which includes:
+We have discovered that the weather data packets received, did not meet our quality standards.
```
Comment (NikosTsiligaridis) [inline docs/rewards/rewards-troubleshooting.mdx:66]: `If you got this error, it means our data evaluation services detected that the received weather packets...`

File: docs/rewards/rewards-troubleshooting.mdx [hunk 19]
```diff
@@ -63,21 +63,39 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
-Weather data packets failed to pass our quality checks. We evaluate all data packets sent by our weather stations using our QoD (Quality of Data) algorithm. You can read more about this algorithm [here](https://docs.weatherxm.com/project/quality-of-data).
-There are several reasons that you may experience this error, which includes:
+We have discovered that the weather data packets received, did not meet our quality standards. 
+To ensure the accuracy of the information sent by our weather stations, we evaluate all data packets using our [Quality of Data (QoD) algorithm](/project/quality-of-data). For more information about this algorithm, please refer to this [here](/project/quality-of-data). 
+
+Below are some of the possible errors and the appropriate actions you can take to resolve the issues:
+
+
+#### Out of Bounds Data 
+It's possible that a sensor sends a completely irrational value, which implies to a faulty sensor. It is recommended that you observe it for a few days before reaching out to our [support team](https://support.weatherxm.com)  for sensor replacement.
```
Comment (NikosTsiligaridis) [inline docs/rewards/rewards-troubleshooting.mdx:73]: `It is recommended that you observe it for a few days before reaching out to our [support team](https://support.weatherxm.com) for sensor replacement.`

If this error persists for multiple days, user must first do a complete power cycle.
1. Remove batteries
2. Cover the solar panel so the station isnt powered by the panel while the batteries are out.
3. Press the reset button a few times to drain all stored power
4. Wait a few minutes
5. Put batteries in

If the above doesn't work, then contact support.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 20]
```diff
@@ -63,21 +63,39 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
 
-- Out of bounds data
-- Spikes on data 
-- Constant values
-- Frozen sensor
 
-Check out our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly,
-and observe it for a couple of days before contacting our support team. If you still receive the error, you are advised to contact our [support team](https://support.weatherxm.com) for more guidance.
+#### Constant Values 
+You should first check your station if it is optimally deployed and then you should contact [support](https://support.weatherxm.com). Take a look at our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly.
```
Comment (NikosTsiligaridis) [inline docs/rewards/rewards-troubleshooting.mdx:81]: Please explain what constant values means (one or more of your sensors report constant values which implies a bad installation, or less often, a problematic sensor).
If your station is optimally installed and the constant value errors persist for a few days, do the above power cycle and if it doesn't go away, contact suhpport.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 21]
```diff
@@ -63,21 +63,39 @@ If the battery is low, the connection between your station and the M5 gateway wi
...
+To ensure the accuracy of the information sent by our weather stations, we evaluate all data packets using our [Quality of Data (QoD) algorithm](/project/quality-of-data). For more information about this algorithm, please refer to this [here](/project/quality-of-data). 
+
+Below are some of the possible errors and the appropriate actions you can take to resolve the issues:
+
+
+#### Out of Bounds Data 
+It's possible that a sensor sends a completely irrational value, which implies to a faulty sensor. It is recommended that you observe it for a few days before reaching out to our [support team](https://support.weatherxm.com)  for sensor replacement.
+
+#### Spikes on Data 
+You should first check your station if it is optimally deployed and then you should contact [support](https://support.weatherxm.com). Take a look at our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly.
```
Comment (NikosTsiligaridis) [inline docs/rewards/rewards-troubleshooting.mdx:76]: a very small explanation first on what data spikes mean "one or more of your sensor are exhibiting irrational spikes"

File: docs/rewards/rewards-troubleshooting.mdx [hunk 22]
```diff
@@ -145,12 +160,25 @@ Please make sure to check the troubleshooting guides depending on your weather s
...
+
+As a result, your station will not be eligible for reward distribution until it is moved back to the declared location or the location is updated by using the "relocate station" function on your mobile app.
+
+:::tip
+When discussing the weather, we always mention a specific location, either directly or indirectly. Therefore, the data provided by weather stations in the WeatherXM Network is meaningless unless we know the location of each station at all times.
+It is crucial for the stability of the Network to have weather stations that can report their current location and remain stationary.
+:::
 
 :::note
 During the beta phase and while the project is still on a test-net, the Proof of Location penalty imposed to devices that went through a relocation is reduced to 2 days for testing purposes
```
Comment (NikosTsiligaridis) [inline docs/rewards/rewards-troubleshooting.mdx:184]: this should be moved below right after
`Please wait 7 days and your weather station will start earning rewards again.`
and rephrased
`During the beta phase and while the project is still on a test-net, the Proof of Location penalty imposed to devices that went through a relocation is reduced to 2 days for testing purposes, however this may change once the project moves to main-net`.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 23]
```diff
@@ -16,7 +16,7 @@ It seems that you haven't connected a wallet to your account yet. Please note th
 1. Learn [how to add or edit a wallet address](/wallet/add-edit-wallet-address).
 2. Then, follow the instructions for [joining Arbitrum Sepolia and see your $WXM tokens](/wallet/how-to-join-arbitrum-and-see-your-tokens#add-arbitrum-sepolia-network-to-metamask).
 
-Once you've completed the above steps, you'll be all set to claim your rewards. Check back tomorrow and see "[How to Claim Rewards](/project/rewards-claiming)" for further guidance.
+Once you've completed the above steps, you'll be all set to claim your rewards. Please wait until the [Quality of Data (QoD) algorithm](/project/quality-of-data) completes. This usually takes until the following day and see "[How to Claim Rewards](/project/rewards-claiming)" for further guidance.
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:19]: This is wrong, it's not the QoD algorithm that produces the "NO_WALLET" error. The user needs to wait for the next run of the reward mechanism.

**Review by PavlosTze — CHANGES_REQUESTED** [review]

File: docs/rewards/rewards-troubleshooting.mdx (hunk 23 above)
Comment (ipanagiv) [reply docs/rewards/rewards-troubleshooting.mdx:19]: fixed

**Review by PavlosTze — APPROVED** [review]

**Review by PavlosTze — CHANGES_REQUESTED** [review]
Hmm something looks broken @ipanagiv , the test deployment failed on this PR:
```
- On source page path = /project/rewards-troubleshooting:
   -> linking to /project/reward-mechanism
```

**Review by PavlosTze — APPROVED** [review]

**Review by NikosTsiligaridis — APPROVED** [review]
